### ISSUE TITLE
Create "claim all" and "claim & stake all" contracts for bonds - Fibonacci: 3

### ISSUE THREAD

**Issue by IceCubeIIIIII** [LobisHQWorkspace/lobis-contracts#5]
**Scenario**
"As a user, I want to claim or claim & stake all my vested bonds in a single transaction to save gas cost".

**Tech**
A couple of buttons could be put on the front end on top of the "my bonds" section.
The transaction would concatenate the transactions of claiming and staking for each bond.

**Reward**
Fibonacci: 3
Bounty: 1 sLOBI

**Comment by IceCubeIIIIII** [comment]
Actually this was also built by Zeus apparently.
Here is the contract:
https://etherscan.io/address/0xE1e83825613DE12E8F0502Da939523558f0B819E

**Comment by julesGoullee** [comment]
> Actually this was also built by Zeus apparently. Here is the contract: https://etherscan.io/address/0xE1e83825613DE12E8F0502Da939523558f0B819E

Good point I'm close. The check for `pendingPayoutFor` is a good idea to implement, `addBondContract` and `removeBondContract` I will prefer my generic function that covers add/remove/update
